packages/components/src/dimension-control/test/index.test.js

```diff
@@ -24,18 +24,15 @@ describe( 'DimensionControl', () => {
 	describe( 'rendering', () => {
 		it( 'renders with defaults', () => {
 			const wrapper = shallow(
-				<DimensionControl
-					instanceId={ uniqueId() }
-					label={ 'Padding' }
-				/>
+				<DimensionControl instanceId={ uuid() } label={ 'Padding' } />
```

Comment:
I also think using a generated id is probably overkill here too. If there's only one dimension control instance in each tree, the instanceId can just be the same value in each test.

Reply:
Good call, updated in https://github.com/WordPress/gutenberg/pull/41751/commits/8c4d4863abedfcadd5ce3d10d1a227ae537fde01 to use the same ID.